ui/src/treeDataProvider/runWithLoadingNotification.ts

```diff
@@ -7,16 +7,24 @@ import { CancellationToken, ProgressLocation, window } from 'vscode';
 import * as types from '../../index';
 import { localize } from '../localize';
 
+let activeCancellationToken: CancellationToken | undefined;
+
 export async function runWithLoadingNotification<T>(context: types.ILoadingTreeContext, callback: (cancellationToken: CancellationToken) => Promise<T>): Promise<T> {
-    return await window.withProgress({ location: ProgressLocation.Notification, cancellable: true }, async (progress, cancellationToken) => {
-        const message: string = context.loadingMessage || localize('loadingAll', 'Loading resources...');
-        const messageDelay: number = context.loadingMessageDelay !== undefined ? context.loadingMessageDelay : 2;
-        const timer: NodeJS.Timer = setTimeout(() => progress.report({ message }), messageDelay * 1000);
+    if (!!activeCancellationToken) {
```

Comment:
I'm still a little concerned because the first callback could finish before the second callback and then the notification is gone and the user has no way to cancel. The reason we added this notification is because it's loading _all_ resources instead of doing our typical pagination with a "load more..." option. That means it could be loading a LOT of resources.

I looked into this more and I don't think we want to display the "Loading resources..." notifications _at all_ in Databases. As mentioned already, it was intended for the case when we're loading _all_ resources. Well the code [here](https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/treeDataProvider/AzExtTreeDataProvider.ts#L179) is using `runWithLoadingNotification` even when `context.loadAll` is false. IMO that's what we should be fixing instead. I believe these are the two lines of code causing the notifications in Databases and I don't see either one using `loadAll`:

https://github.com/microsoft/vscode-cosmosdb/blob/master/src/postgres/commands/registerPostgresCommands.ts#L57

https://github.com/microsoft/vscode-cosmosdb/blob/master/src/mongo/registerMongoCommands.ts#L137